lightning/src/chain/onchaintx.rs

```diff
@@ -1036,8 +1036,10 @@ impl<ChannelSigner: WriteableEcdsaChannelSigner> OnchainTxHandler<ChannelSigner>
 			}
 		}
 		for ((_package_id, _), ref mut request) in bump_candidates.iter_mut() {
+			// `height` is the height being disconnected, so our `current_height` is 1 lower.
```

Comment:
Trying to claim at next block connection works well, until a user calls `invalidateblock` and break this assumption, I think (though it’s a quite unorthodox uses of the consensus engine).